go-controller/pkg/ovn/pods.go

```diff
@@ -459,10 +459,11 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) (err error) {
 	}
 
 	// Ensure the namespace/nsInfo exists
-	routingExternalGWs, routingPodGWs, err := oc.addPodToNamespace(pod.Namespace, podIfAddrs)
+	routingExternalGWs, routingPodGWs, ops, err := oc.addPodToNamespace(pod.Namespace, podIfAddrs)
```

Comment:
the `ops` we get back here is a bit weird, I'd expect a value always even if the transact below results in a no-op. But that's not the case, sometimes we are not getting any ops returned.